app/controllers/reports_controller.rb

```diff
@@ -0,0 +1,42 @@
+# frozen_string_literal: true
+
+class ReportsController < BaseTraineeController
+
+  def index
+    @current_user = current_user
+  end
+
+  def itt_new_starter_data_signoff
+    respond_to do |format|
+      format.html
+      format.csv do
+        authorize(:trainee, :export?)
+        send_data(data_for_export.data, filename: "Itt-new-starter-data-signoff-2022.csv", disposition: :attachment)
+      end
+    end
+  end
+
+  private
+  def data_for_export
+    @data_export ||= Exports::TraineeSearchData.new(policy_scope(filtered_trainees))
+  end
+
+  def filtered_trainees
+    #will need to be made adaptive to year
+    census_date = date_of_nth_weekday(10, 2022 ,3 ,2)
+    Trainee.where("itt_start_date <= :date or commencement_date <= :date", date: census_date)
+           .or(Trainee.where(commencement_date: nil))
+           .where(start_academic_cycle_id: AcademicCycle.current)
+           .where.not(state: 0)
```

Comment:
I think this would end up including trainees from hesa that have gone through the TRN request but not been registered - which we wouldn't want. Another reason we'd be better off having a simple state registered/not-registered rather than needing tricky conditions.

Reply:
From what Abi said on Slack, they should already be excluded.

Reply:
They'll be excluded if you use the `Trainee::Filter` -> this is what we use to find trainees for the two main pages, the existing export and the home page counts because it removes 'empty' trainees and now the TRN data trainees.